From 6a4d7f91407e69eb930b57ba0a79bb8758c8c2c4 Mon Sep 17 00:00:00 2001
From: The-going <48602507+The-going@users.noreply.github.com>
Date: Sun, 29 May 2022 23:02:24 +0300
Subject: [PATCH 5/5] Fix duplicate nodes for sun50i-h5-orangepi-pc2

The problem with duplicate nodes in the device tree is that
the first one in the queue will be applied. The subsequent
one will be ignored, not replaced.
---
 .../dts/allwinner/sun50i-h5-orangepi-pc2.dts  | 20 -------------------
 1 file changed, 20 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts b/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
index 953d5cdf1..20bcbe707 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h5-orangepi-pc2.dts
@@ -99,10 +99,6 @@ &codec {
 	status = "okay";
 };
 
-&cpu0 {
-	cpu-supply = <&reg_vdd_cpux>;
-};
-
 &de {
 	status = "okay";
 };
@@ -182,22 +178,6 @@ &ohci3 {
 	status = "okay";
 };
 
-&r_i2c {
-	status = "okay";
-
-	reg_vdd_cpux: regulator@65 {
-		compatible = "silergy,sy8106a";
-		reg = <0x65>;
-		regulator-name = "vdd-cpux";
-		silergy,fixed-microvolt = <1100000>;
-		regulator-min-microvolt = <1000000>;
-		regulator-max-microvolt = <1400000>;
-		regulator-ramp-delay = <200>;
-		regulator-boot-on;
-		regulator-always-on;
-	};
-};
-
 &sound_hdmi {
 	status = "okay";
 };
-- 
2.35.3

